scanner/src/routes/q/[iss]/[product]/[id]/+page.svelte

```diff
@@ -0,0 +1,235 @@
+<script lang="ts">
+  import Header from "$components/Header/index.svelte"
+  import Footer from "$components/Footer/index.svelte"
+  import BatteryPng from "$assets/battery.png"
+  import GreenCheckSvg from "$assets/green-check.svg"
+  import ChevronLeftSvg from "$assets/chevron-left.svg"
+  import { onMount } from "svelte"
+  import { page } from "$app/stores"
+
+  let data: any = {}
+  let availableProductFetched: boolean = false
+  let secondProductFetched: boolean = false
+
+  onMount(async () => {
+    data = history.state
+    console.log(data)
+  })
+</script>
+
+<Header />
+<div class="content">
+  <div class="logo-title">
+    <div class="logo"><img src={BatteryPng} alt="" aria-hidden="true" /></div>
+    <div class="title">{data.name}</div>
+  </div>
+  <div class="card">
+    <div class="basic-information">BASIC INFORMATION</div>
```

Comment:
Please use `text-transform: uppercase` instead: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
```suggestion
    <div class="basic-information">Basic information</div>
```